src/test/java/step1/StringTest.java

```diff
@@ -0,0 +1,66 @@
+package step1;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class StringTest {
+
+    @Test
+    @DisplayName("1,2을 ,로 split 했을 때 1과 2로 분리된다.")
+    void split1() {
+        String text = "1,2";
+
+        String[] actual = text.split(",");
+        String[] expected = {"1", "2"};
+
+        assertThat(actual).isEqualTo(expected);
```

Comment:
```suggestion
        String[] actual = "1,2".split(",");
        assertThat(actual).contains("1", "2");
```
굳이 로컬 변수를 빼지 말고 위와 같이 구현하는 것은 어떨까?
로컬 변수로 분리한 의미가 없어 보인다.